bin/git-merge-branch

```diff
@@ -2,13 +2,25 @@
 
 set -e
 
+if git symbolic-ref --short refs/remotes/origin/HEAD >/dev/null; then
+  main_branch="$(git symbolic-ref --short refs/remotes/origin/HEAD | sed 's@^origin/@@')" "$@"
+else
+  echo "You don't have a primary branch reference set for your origin remote.
+  Use:
+  git symbolic-ref refs/remotes/origin/HEAD refs/remotes/origin/{name_of_your_primary_branch}
+
+  to set the reference and then try merging again."
+
+  exit
```

Comment:
Should this `exit 1` as well in case someone's grouped operations together with `&&`?

Reply:
Good catch! I don't shell script very often...